game-core/src/main/java/games/strategy/triplea/ui/screen/TileManager.java

```diff
@@ -110,21 +145,12 @@ public TileManager(final UiContext uiContext) {
     }
   }
 
-  public List<Tile> getTiles() {
-    acquireLock();
-    try {
-      return new ArrayList<>(tiles);
-    } finally {
-      releaseLock();
-    }
-  }
-
   private void acquireLock() {
-    LockUtil.INSTANCE.acquireLock(lock);
+    Tile.LOCK_UTIL.acquireLock(lock);
```

Comment:
Tile.LOCK_UTIL is a static reference to LockUtil.INSTANCE, so this can be kept as well